.lein-yagni

```diff
@@ -1 +1,3 @@
 api.handler/entry
+rules.abilities/row-affinity
+rules.abilities/weaken
```

Comment:
This being necessary feels like a bad sign...

Reply:
Maybe you could create an issue explaining why it feels like a bad sign, so it can be checked and changed if needed?

Reply:
I might do that if I ever remember what I was thinking :upside_down_face:

Reply:
Oh! This is the "You aren't gonna need it" ignore file... there ideally should be *nothing* here except for the application entry points (so only `api.handle/entry` so far)